.github/workflows/dandi_release.yml

```
@@ -1,6 +1,9 @@
-name: Test against latest DANDI release
+name: Test against latest DANDI release - no network
 on: workflow_call
 
+env:
+  DANDI_TESTS_NONETWORK: "1"
```

Maybe it's just me, but at first I always read as "none twork" 😅 
```suggestion
  DANDI_TESTS_NO_NETWORK: "1"
```

---

While I completely agree - this is the name of the environment variable that must be set to have this effect on the DANDI testing suite, so I cannot change it here

---

ah okay, thanks for the clarification!